src/main/resources/application.properties

```diff
@@ -1,4 +1,5 @@
-management.endpoints.web.exposure.include=info,health
+management.endpoint.prometheus.enabled=false
```

Comment:
Add `true` to `docker-compose.yml` in repo `eth-compose`.

Reply:
Already did that in `docker-compose.override.yml`